libethdrivers/CMakeLists.txt

```diff
@@ -63,7 +63,7 @@ mark_as_advanced(
 )
 add_config_library(ethdrivers "${configure_string}")
 
-if("${KernelPlatform}" MATCHES "imx8mq-evk")
+if(("${KernelPlatform}" MATCHES "imx8mq-evk") OR ("${KernelPlatform}" MATCHES "imx8mm-evk"))
```

Comment:
I just wonder, was there any reason we say `MATCHES` here and not `STREQ`? We could simply use the flags here also if this selects well define platforms: `if(KernelPlatformImx8mq-evk OR KernelPlatformImx8mm-evk)`